report-ng/app/src/components/timings/test-timings/test-timings.ts

```diff
@@ -194,24 +193,32 @@ export class TestTimings extends AbstractViewModel {
                             return "";
                         }
 
-                        const testNames = this._bars[dataIndex].methodList.map(method => method.name);
-                        let tooltipString = `${testNumber} test case(s): <br>`;
+                        let tooltipString = `${testNumber} test case(s): <br/><br/>`;
 
                         if (testNumber < TestTimings.TEST_NUMBER_LIMIT) {
-                            tooltipString += "<ul>";
-                            testNames.forEach(testCase => {
-                                tooltipString += `<li>${testCase}</li>`;
+                            this._bars[dataIndex].methodList.map(method => {
+                                tooltipString += `<div class="mb1"> 
+                                                    <span class="ml1 mr1 status-badge badge__dense tag__dense status-${this._statusConverter.getClassForStatus(method.status)}">
+                                                    ${this._statusConverter.getClassForStatus(method.status)} </span>`
+
+                                tooltipString += `${method.name}`
+
+                                if(method.methodType==2) tooltipString += `<span class="ml1 mr1 config-badge badge__dense tag tag__dense"> Configuration </span>`;
```

Comment:
Improve your `if`:
* Please use `MethodType.CONFIGURATION_METHOD` instead of numbers if possible
* Please always use `{...}`